src/installation/index.md

```diff
@@ -35,7 +35,7 @@ from [other mirrors](../xbps/repositories/mirrors/index.md). Previous releases
 can be found under <https://alpha.de.repo.voidlinux.org/live/>, organized by
 date.
 
-### Verifying images
+## Verifying images
```

Comment:
Why would you say image verification is not part of downloading the image?

Reply:
Well, you first download an image, and then you verify it. It's a separate step. And one that needs way more explanation than downloading. So, I'd say it really deserves to be a separate section. Doing it this way also makes the layout work with fewer heading levels, which, IMO, is appropriate in this case.